src/server/ApiRoutes/Hold.js

```diff
@@ -118,13 +118,15 @@ function getDeliveryLocations(barcode, patronId, accessToken, cb, errorCb) {
     }
   )
   .then(barcodeAPIresponse => {
-    const deliveryLocationWithAddress = mapLocationDetails(
-      barcodeAPIresponse.data.itemListElement[0].deliveryLocation
-    );
+    const eddRequestable = (barcodeAPIresponse.data.itemListElement[0].eddRequestable) ?
```

Comment:
OK I don't think it works or at least works as the way I think lol
I got a 500 if I put two barcodes in the endpoint.